Makefile

```diff
@@ -12,6 +12,23 @@ BEAM_VERSION = 2.37.0
 UBUNTU_IMAGE = ubuntu@sha256:9101220a875cee98b016668342c489ff0674f247f6ca20dfc91b91c0f28581ae
 # prognostic base image is updated manually, not on every commit
 PROGNOSTIC_BASE_VERSION = 1.0.0
+DOCKER_AUTH_ARGS = \
+	-v ${GOOGLE_APPLICATION_CREDENTIALS}:/tmp/key.json \
+	-e GOOGLE_APPLICATION_CREDENTIALS=/tmp/key.json \
+	-e FSSPEC_GS_REQUESTER_PAYS=vcm-ml
+
+DOCKER_INTERACTIVE_ARGS = \
```

Comment:
I could put these below in the one make target where they're used, it just felt "cleaner" this way.